packages/protocol/utils/generate_genesis/taikoL2.ts

```diff
@@ -464,12 +464,6 @@ async function generateContractConfigs(
                 _paused: 1, // _FALSE
                 // Ownable2Upgradeable
                 _owner: ownerSecurityCouncil,
-                authorizedAddresses: {
```

Comment:
@davidtaikocha please check if this is the right modification. AuthorizableContract.sol is removed, and SignalService now needs an AddressMamager to init.